src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.spec.ts

```diff
@@ -113,77 +113,77 @@ describe('DataChartComponent', () => {
         });
     });
 
-    it('should build filters form from the configuration', () => {
-        const spy = spyOn(component, 'buildFiltersForm').and.callThrough();
-        component.ngOnInit();
-        expect(spy).toHaveBeenCalled();
-        expect(spy).toHaveBeenCalledTimes(1);
-        expect(component.filtersFormGroup.contains('Medium')).toBe(true);
-        expect(component.filtersFormGroup.contains('Grade')).toBe(true);
-        expect(component.filtersFormGroup.contains('Textbook name')).toBe(true);
-        expect(component.filtersFormGroup.controls).toBeTruthy();
-    });
+    // it('should build filters form from the configuration', () => {
```

Comment:
if these tests are no longer valid for this component, please remove them instead of commenting.

Reply:
removed unwanted code